specification/sdk-tracing.md

```diff
@@ -110,8 +116,18 @@ invocations. The span processors are invoked only when
 must be used to implement [span exporter](#span-exporter) to batch and convert
 spans.
 
-Span processors can be registered directly on SDK Tracer and they are invoked in
-the same order as they were registered.
+Span processors can be registered directly on SDK `TracerFactory` and they are
+invoked in the same order as they were registered.
+
+All `Tracer` instances created by a `TracerFactory` share the same span processors.
+Changes to this collection reflect in all `Tracer` instances.
```

Comment:
```suggestion
Changes to this collection reflects in all `Tracer` instances.
```

Reply:
It's "change**s**", so reflect without "s" seems correct.